networkit/cpp/graph/test/SpanningGTest.cpp

```diff
@@ -32,7 +32,7 @@ TEST_F(SpanningGTest, testKruskalMinSpanningForest) {
     }
 }
 
-TEST_F(SpanningGTest, testKruskalMinSpanningForestSimple) {
+TEST_F(SpanningGTest, testKruskalMinSpanningForestSimple1) {
```

Comment:
Please use a more representative name for these two tests. For example, in this test the MFS is a MST since all edges have the same weight so we could call it `testKruskalMinimumSpanningForestIsMinimumSpanningTree` or `testKruskalMinimumSpanningForestUnweightedGraph`.

For the second test, we could just test `getTotalWeight` since the correctness of the algorithm is already tested above; so I suggest to remove the last part of the test and name it `testKruskalMinimumSpanningForestGetTotalWeight`.